### ISSUE TITLE
Key derivation is failing when attempting to load from keystore.

### ISSUE THREAD

**Issue by Ed-Commodum** [icon-project/icon-sdk-js#71]
When exporting a keystore from a newly generated wallet and then immediately importing it into a new wallet, the import is failing with
```[WALLET ERROR] Key derivation is failed (possibly wrong passphrase).```

Here is the test case:
```
const IconService = require('icon-sdk-js');

const main = () => {

    const generatedWallet = IconService.Wallet.create();
    console.log(generatedWallet.getAddress());
    
    const pass = "badPassword";
    const keystore = generatedWallet.store(pass);
    
    const importedWallet = IconService.Wallet.loadKeystore(keystore, pass);
    console.log(importedWallet.getAddress());

};

main();
```

Node Version: `18.14.2`
icon-sdk-js Verison: `1.4.4` 

Additional output when running with `--trace-uncaught` flag:
```hrown at:
    at loadKeystore (/root/icon-sdk-js-keystore-test/node_modules/icon-sdk-js/build/icon-sdk-js.node.min.js:2:183385)
    at main (/root/icon-sdk-js-keystore-test/index.js:11:47)
    at /root/icon-sdk-js-keystore-test/index.js:16:1
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at executeUserEntryPoint (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47```

**Comment by Ed-Commodum** [comment]
I rolled back the version of the sdk and the most recent version that does not exhibit this bug is `1.4.1`.

**Comment by inwonkim** [comment]
@Ed-Commodum Thank you for reporting the bug. We've released a fixed version(1.4.5).
